ui/pages/confirmations/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -259,21 +256,11 @@ export default class ConfirmTransactionBase extends Component {
       }
     }
 
-    if (hexMaximumTransactionFee !== prevHexMaximumTransactionFee) {
-      fetchEstimatedL1Fee(txData?.chainId, txData)
-        .then((result) => {
-          this.setState({
-            estimatedL1Fees: result,
-          });
-        })
-        .catch((_err) => {
-          this.setState({
-            estimatedL1Fees: 0,
-          });
-        });
-      if (useMaxValue) {
-        this.updateValueToMax();
-      }
+    if (
+      hexMaximumTransactionFee !== prevHexMaximumTransactionFee &&
+      useMaxValue
+    ) {
+      this.updateValueToMax();
```

Comment:
Do we need to update the send max logic to also subtract the new L1 fee property or is that already done?

Reply:
This may be a pre-existing bug we can address in a different PR.